games-emulation/ryujinx/ryujinx-1.1.1221.ebuild

```diff
@@ -0,0 +1,404 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DOTNET_PKG_COMPAT=8.0
+NUGETS="
+avalonia@11.0.7
+avalonia.angle.windows.natives@2.1.0.2023020321
+avalonia.buildservices@0.0.29
+avalonia.controls.colorpicker@11.0.4
+avalonia.controls.colorpicker@11.0.7
+avalonia.controls.datagrid@11.0.7
+avalonia.controls.itemsrepeater@11.0.4
+avalonia.desktop@11.0.7
+avalonia.diagnostics@11.0.7
+avalonia.freedesktop@11.0.7
+avalonia.markup.xaml.loader@11.0.7
+avalonia.native@11.0.7
+avalonia.remote.protocol@11.0.4
+avalonia.remote.protocol@11.0.7
+avalonia.skia@11.0.0
+avalonia.skia@11.0.4
+avalonia.skia@11.0.7
+avalonia.svg@11.0.0.13
+avalonia.svg.skia@11.0.0.13
+avalonia.themes.simple@11.0.7
+avalonia.win32@11.0.7
+avalonia.x11@11.0.7
+commandlineparser@2.9.1
+concentus@1.1.7
+discordrichpresence@1.2.1.24
+dynamicdata@8.3.27
+excss@4.2.3
+fluentavaloniaui@2.0.5
+fsharp.core@7.0.200
+gtksharp.dependencies@1.1.1
+harfbuzzsharp@2.8.2.3
+harfbuzzsharp@7.3.0
+harfbuzzsharp.nativeassets.linux@2.8.2.3
+harfbuzzsharp.nativeassets.linux@7.3.0
+harfbuzzsharp.nativeassets.macos@2.8.2.3
+harfbuzzsharp.nativeassets.macos@7.3.0
+harfbuzzsharp.nativeassets.webassembly@2.8.2.3
+harfbuzzsharp.nativeassets.webassembly@7.3.0
+harfbuzzsharp.nativeassets.win32@2.8.2.3
+harfbuzzsharp.nativeassets.win32@7.3.0
+libhac@0.19.0
+microcom.codegenerator.msbuild@0.11.0
+microcom.runtime@0.11.0
+microsoft.aspnetcore.app.runtime.linux-x64@8.0.2
+microsoft.aspnetcore.app.runtime.osx-x64@8.0.2
+microsoft.aspnetcore.app.runtime.win-x64@8.0.2
+microsoft.codeanalysis.analyzers@3.0.0
+microsoft.codeanalysis.analyzers@3.3.4
+microsoft.codeanalysis.common@3.8.0
+microsoft.codeanalysis.common@4.8.0
+microsoft.codeanalysis.csharp@3.8.0
+microsoft.codeanalysis.csharp@4.8.0
+microsoft.codeanalysis.csharp.scripting@3.8.0
+microsoft.codeanalysis.scripting.common@3.8.0
+microsoft.codecoverage@17.9.0
+microsoft.csharp@4.3.0
+microsoft.csharp@4.7.0
+microsoft.dotnet.platformabstractions@3.1.6
+microsoft.extensions.dependencymodel@6.0.0
+microsoft.identitymodel.abstractions@7.3.0
+microsoft.identitymodel.jsonwebtokens@7.3.0
+microsoft.identitymodel.logging@7.3.0
+microsoft.identitymodel.tokens@7.3.0
+microsoft.io.recyclablememorystream@3.0.0
+microsoft.netcore.app.host.osx-x64@8.0.2
+microsoft.netcore.app.host.win-x64@8.0.2
+microsoft.netcore.app.runtime.linux-x64@8.0.2
+microsoft.netcore.app.runtime.osx-x64@8.0.2
+microsoft.netcore.app.runtime.win-x64@8.0.2
```

Comment:
Those are bound to the .NET SDK version.

Since we are on Gentoo and support only Linux builds for .NET those shoudl be removed.

.csproj files can be modded to not require those, in case of update of the .NET SDK (say from 8.0.2 -> 8.1.0) ryujinx build will fail casue it will have only 8.0.2.

Those are most likely pulled in by building for a "wrong" RID, which could have been hardcoded into projects' csproj files.